### PR TITLE
Closed #2435 fixed create new map for openlayers and cesium

### LINKED ISSUE #2435: Still a default Leaflet map if you open a New Map
### Description
If you open a New Map in Mapstore 2 you still open a leaflet map (even though you change the "Select Map Viewer" the new map is still a Leaflet map).
To work around that problem you can modify the link of the new map created from "https://dev.mapstore2.geo-solutions.it/mapstore/#/viewer/leaflet/new" to "https://dev.mapstore2.geo-solutions.it/mapstore/#/viewer/openlayers/new" and reload the page.

### Please indicate if this issue is related to a bug or a new feature request
- [x] Bug
- [ ] New Feature

*Browser Affected*
- [x] Internet Explorer
- [x] Chrome
- [x] Firefox
- [ ] Safari

*Browser Version Affected*
All.

*Steps to reproduce*
- Modify the Select Map Viewer from "Leaflet" to "OpenLayers"
- open a new map.

*Expected Result*
- The new map has a OpenLayer viewer if "OpenLayers" is selected.

*Current Result*
- The new map is still a Leaflet map even through "OpenLayers" is selected

### PR DESCRIPTION
**Pull request by MV88**
## Description
It is possible to create new map with all three maptypes

## Issues
 - Fix #2435

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

**What is the current behavior?** (You can also link to an open issue here)
#2435

**What is the new behavior?**
it opens new map with the selected maptype

**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [x] No

**Other information**:
Be sure to include **maptype** reducer in app.jsx for every project after this change

### CODE REVIEW

File: web/client/plugins/Maps.jsx [hunk 1]
```diff
@@ -1,24 +1,24 @@
-const PropTypes = require('prop-types');
 /**
- * Copyright 2016, GeoSolutions Sas.
- * All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
+* Copyright 2016, GeoSolutions Sas.
```
Comment (mbarto) [inline web/client/plugins/Maps.jsx]: 2017

**Comment by mbarto** [comment]
I cannot see neither tests or docs

**Comment by MV88** [comment]
@offtherailz the reducer is included in the [app.jsx](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/product/app.jsx#L29) and in the [maps](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/plugins/Maps.jsx#L146) plugin.

For the projects, we need to include the reducer in the app.jsx, or just the maps plugin. 
By default it returns leaflet if it does not find anything in the state.
IMHO, the maptype should stay in one piece of the state and not spread it in different places, that's why i'm searching for it only in the maptype reducer.
Should we open an issue on our projects for updating this if needed?

**Comment by MV88** [comment]
@mbarto i cannot add test for plugins

**Pull request merged**